lib/rpmsg/rpmsg_virtio.c

```diff
@@ -358,6 +358,19 @@ static void rpmsg_virtio_release_rx_buffer(struct rpmsg_device *rdev,
 	metal_mutex_release(&rdev->lock);
 }
 
+static int rpmsg_virtio_notify_wait(struct rpmsg_virtio_device *rvdev,
+				    struct virtqueue *vq)
+{
+	struct virtio_vring_info *vring_info;
+
+	vring_info = &rvdev->vdev->vrings_info[vq->vq_queue_index];
+
+	if (rvdev->notify_wait_cb)
+		return rvdev->notify_wait_cb(&rvdev->rdev, vring_info->notifyid);
+
+	return RPMSG_EOPNOTSUPP;
+}
```

Comment:
Nitpicking... Could you inverse the logic
```
static int rpmsg_virtio_notify_wait(struct rpmsg_virtio_device *rvdev,
				    struct virtqueue *vq)
{
	struct virtio_vring_info *vring_info;

	vring_info = &rvdev->vdev->vrings_info[vq->vq_queue_index];

	if (!rvdev->notify_wait_cb)
		return RPMSG_EOPNOTSUPP;

	return rvdev->notify_wait_cb(&rvdev->rdev, vring_info->notifyid);
}
```

Reply:
Yes, done